.github/workflows/test.yml

```diff
@@ -9,18 +9,18 @@ jobs:
   test:
     environment: staging
     runs-on: ubuntu-latest
+    name: Ruby ${{ matrix.ruby-version }}
     strategy:
       matrix:
-        ruby-version: [2.5, 2.6, 2.7, 3.0]
+        ruby-version: [2.5, 2.6, 2.7, "3.0"]
```

Comment:
Very subtle, nice!

While .yml won't trip up rubocop, could we nevertheless use single quotes here (for non-interpolated values) for consistency's sake?